vision_model/model_base.py

```diff
@@ -0,0 +1,32 @@
+from typing import Dict, List, Union
+import torch
+from .plugin_base import PluginBase
+
+
+class ModelBase(torch.nn.Module):
```

Comment:
Should we emphasize this is the training interface?

Reply:
The current design is trying to provide both training and inference features in a single interface. We need to discuss whether we should split it into two.

Reply:
It would be cleaner to have a dedicated inference interface, and it should be very straight-forward.

Some models/operations like captioning won't be easy to be onboarded for training, but we still want to support inference for benchmark

Reply:
ok, I'll split it to two interfaces.